Shopery/libs/src/lib/Button/Button.module.scss

```
@@ -0,0 +1,48 @@
+.button{
```

seems like formatting for scss does not work

---

I mean that there are no whitespace
```suggestion
.button {
```